modules/storages/app/components/storages/admin/oauth_client_info_component.html.erb

```diff
@@ -0,0 +1,36 @@
+<%=
+  grid_layout('op-storage-view--row', tag: :div, align_items: :center) do |grid|
+    grid.with_area(:item, tag: :div, mr: 3) do
+      concat(
+        render(
+          Primer::Beta::Text.new(font_weight: :semibold, mr: 1, test_selector: 'storage-oauth-client-label')
+        ) { I18n.t("storages.file_storage_view.#{storage.short_provider_type}_oauth") }
+      )
+      concat(configuration_check_label_for(:storage_oauth_client_configured))
+    end
+
+    grid.with_area(:description, tag: :div, color: :subtle, test_selector: 'storage-oauth-client-id-description') do
+      render(Primer::Beta::Truncate.new(font_weight: :light)) { provider_oauth_client_description }
```

Comment:
This looks a bit different than specified in figma due to the `font_weight`. I noticed that "automatically managed folder" form as well.